Mathlib/NumberTheory/Modular.lean

```diff
@@ -508,4 +513,29 @@ end UniqueRepresentative
 
 end FundamentalDomain
 
+section UpperHalfPlane
+
+lemma exists_translate (τ : ℍ) :
+    ∃ γ : SL(2, ℤ), 1 / 2 ≤ im (γ • τ) := by
+  obtain ⟨γ, hγ⟩ := ModularGroup.exists_smul_mem_fd τ
+  use γ
+  nlinarith [ModularGroup.three_le_four_mul_im_sq_of_mem_fd hγ, UpperHalfPlane.im_pos (γ • τ)]
+
+/--For every `τ : ℍ` there is some matrix in `SL(2, ℤ)` that sends it to an elemenet whose
```

Comment:
```suggestion
/-- For every `τ : ℍ` there is some matrix in `SL(2, ℤ)` that sends it to an element whose
```
and I would either change the second line to "and whose denominator has norm at most 1", or say what \gamma is.